playwright.config.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2016-2023 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './tests',
```

Comment:
I will look into this. I think it puts them in a folder that matches the test filename so that snapshots from different test files are separate.

Reply:
The value `./test/snapshots` is not valid for `snapshotPathTemplate`. We'd need a template like `./test/snapshots/{platform}/{projectName}/{testFilePath}/{arg}{ext}`.

I think it would be good to move the `platform` and `project` to folders (instead of part of the filename). I don't think it's a good idea to remove the test file name, though.

Reply:
Any values we want to keep can be added to the filename. The platform, project name, or test file path aren't beneficial, given that we only have one test on a single platform/project. 

We can revisit this if we need to in the future.

Reply:
We can add interaction tests with a simplified name that doesn't include the lengthy file path.

Reply:
I am not sure what you mean. I think we need to meet about this.

Reply:
I shortened the snapshot file paths by removing the `platform` and `project` and shortening the test filename.

Reply:
I did not realize that Playwright doesn't allow duplicate test names, so the test filename isn't important like I thought. I assumed they included it by default to prevent the problem I mentioned, but I am not sure.

Reply:
I like the idea of using the component name as a folder. I think I have it working.

Reply:
The snapshot file path is now `./tests/snapshots/{component}/{story id or test title}.png`.

Reply:
It's detecting the component name incorrectly. `storyId.split('-')[0]` doesn't work for `stack-view`, etc. I am working on a fix.

Reply:
Okay, that is fixed.